tools/winmd2objc/UnitTests/UnitTests.vcxproj

```diff
@@ -22,7 +22,8 @@
     <ProjectGuid>{1782D50C-E6B5-4499-90B6-B72AED2FBEDA}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>UnitTests</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
+    <TargetPlatformVersion>10.0.15063.0</TargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
```

Comment:
Do both this and the below project need a Min version, or is it okay without one?

Reply:
It seems fine without it. The build machine builds have passed beyond tools.